Objects/unicodeobject.c

```diff
@@ -15120,15 +15120,13 @@ unicode_subtype_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 
 PyDoc_STRVAR(unicode_doc,
 "str(object='') -> str\n\
-str(bytes_or_buffer[, encoding[, errors]]) -> str\n\
+str(object, encoding='utf-8', errors='strict') -> str\n\
```

Comment:
I'm not seeing why this needs two signatures now, `str(object='', encoding='utf-8', errors='strict')` would suffice.

Reply:
I don’t think that merging the two lines is correct.